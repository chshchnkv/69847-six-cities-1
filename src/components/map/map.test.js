import React from "react";
import renderer from "react-test-renderer";
import Map from "./map";

const mock = {
  city: {
    longitude: 52.38333,
    latitude: 4.9
  },
  zoom: 12,
  pins: [
    {
      longitude: 52.3909553943508,
      latitude: 4.85309666406198
    },
    {
      longitude: 52.369553943508,
      latitude: 4.85309666406198
    },
    {
      longitude: 52.3909553943508,
      latitude: 4.929309666406198
    },
    {
      longitude: 52.3809553943508,
      latitude: 4.939309666406198
    },
    {
      longitude: 52.3809553943508,
      latitude: 4.929309666406198
    }
  ]
};

it(`Map renders correctly`, () => {
  const tree = renderer.create(<Map city={mock.city} zoom={mock.zoom} pins={mock.pins}/>);
  expect(tree).toMatchSnapshot();
});
