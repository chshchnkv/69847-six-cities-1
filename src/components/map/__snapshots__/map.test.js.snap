// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renders correctly 1`] = `
<div
  id="map"
  style={
    Object {
      "display": "flex",
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
